modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/WorkflowMetricsAllItemsList.path

```diff
@@ -20,6 +20,11 @@
 	<td>//p[span[contains(text(),'${key_rowName}')] and contains(.,'${key_itemName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SELECT_TRACK_WORKFLOW_LINK_PROCESS_STATUS</td>
```

Comment:
In the path, you should not name it by the action provided, but by the element it represents. The track approval link it's a element. 
This path has already been merged as: `TRACK_WORKFLOW_LINK`